pkg/driver/driver_aws.go

```diff
@@ -167,20 +177,23 @@ func (d *AWSDriver) generateBlockDevices(blockDevices []v1alpha1.AWSBlockDeviceM
 		if disk.DeviceName == "/root" || len(blockDevices) == 1 {
 			deviceName = *rootDeviceName
 		}
+		deleteOnTermination := disk.Ebs.DeleteOnTermination
 		volumeSize := disk.Ebs.VolumeSize
 		volumeType := disk.Ebs.VolumeType
-		deleteOnTermination := disk.Ebs.DeleteOnTermination
 		encrypted := disk.Ebs.Encrypted
 
 		blkDeviceMapping := ec2.BlockDeviceMapping{
 			DeviceName: aws.String(deviceName),
 			Ebs: &ec2.EbsBlockDevice{
-				DeleteOnTermination: aws.Bool(deleteOnTermination),
-				Encrypted:           aws.Bool(encrypted),
-				VolumeSize:          aws.Int64(volumeSize),
-				VolumeType:          aws.String(volumeType),
+				Encrypted:  aws.Bool(encrypted),
+				VolumeSize: aws.Int64(volumeSize),
+				VolumeType: aws.String(volumeType),
 			},
 		}
+		if deleteOnTermination != nil {
```

Comment:
This isn't needed, is it?

Reply:
This is just to be sure, and could otherwise result in unexpected behavior from AWS.